### ISSUE TITLE
visualization tool does not work?

### ISSUE THREAD

**Issue by RDEVIDTS** [hl7-be/lab#243]
I think, the visualization tool does not work.

**Comment by bdc-ehealth** [comment]
@RDEVIDTS ,

you are right, there are still problems with tx.fhir.org, the international FHIR terminology server.

**Comment by bdc-ehealth** [comment]
@RDEVIDTS ,

there was some additional problem, which we have solved now.
